<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FtpConnectDlg</class>
 <widget class="QDialog" name="FtpConnectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>222</width>
     <height>125</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_host">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_Host"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_user">
        <property name="text">
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_User"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_pass">
        <property name="text">
         <string>Pass:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_Pass"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_connect">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
